### DISCUSSION TITLE
Meeting Agenda 12.04.2024

### DISCUSSION THREAD

**Discussion by Marc738** [SE-TINF22B2/G3-ApeRepublic#103] [Protokolle]
* Wann/Wie sollen Meetings ablaufen? (Kontept)
* Extra Meeting für Front- & Backend (Subteams)?
* Rollenverteilung im Team (Meeting Scheduler, Dokumentator
* Kernfunktionen definieren (Abstufung wie entwickelt wird)

* Meilensteine setzen
* Zeitplanung mit Berücksichtigung von Abschlusspräsentation und Gesamtdokumentation

**Comment by elnaz-gharoon** [comment]
Kann ich in Diskussion, ein General Category eröffnen und meine Zeit darin erfassen? 
Abstimmung über die nächsten Schritten und wie können wir die reste Aufgaben erledigen
Nächste Report zusammen beim ein Gespräch am Dientsag abstimmen und zusammen schreiben 
Wann ist genau die Abschlusspräsentation für unsrere Gruppe ? am 19.Juni or 12.Juni ?

**Reply by elnaz-gharoon** [reply]
MeilenStones for Angular FE
Ich soll zuerst folgendes Page entwickeln dann SignUp Page & Forget Password Page
![image](https://github.com/SE-TINF22B2/G3-ApeRepublic/assets/90132658/f7154da4-dc07-49aa-8176-bc62431b5475)

**Comment by culmets** [comment]
Wann/Wie sollen Meetings ablaufen? (Konzept)

Nach SE Vorlesung in der Pause absprechen wie es für die Woche weiter geht und dann einen regelmäßigen Austausch Termin, z. B. Donnerstags oder Dienstags Abends. Von Freitag mittag an bitte nichts mehr
"-----"
Extra Meeting für Front- & Backend (Subteams)?

Nach Bedarf? Marc am Montag nach Mathe oder am Dienstag zwischen den Vorlesungen?
"-----"
Rollenverteilung im Team (Meeting Scheduler, Dokumentator

Matej hatte gesagt er kann Doku machen? Und Marc Meeting Fee? 
Was noch für Rollen?
"-----"
Kernfunktionen definieren (Abstufung wie entwickelt wird)

- Datenbank aufsetzen
- Statische Aktien Metadaten reinladen
- Abfragen definieren? Die die immer wieder gebraucht werden (welche aktien hat der typ, wie das leaderboard berechnet wird
- Abfragen implementieren
-

**Reply by culmets** [reply]
Milestones für die DB: DB ist aufgesetzt | DB ist mit Daten gefüllt
Milestones für Java Backend:  JPA ist implementiert
   genauer: Tabellen haben JPA Klassen als Gegenstück, SQL Abfragen in Java implementiert.

**Reply by MatejEvc** [reply]
Schreibst du dann den Java Code für die Kommunikation mit der DB?
Zusammen mit SQL Abfragen?
@culmets

**Reply by MatejEvc** [reply]
Komponentendiagramme erstellen, Klassendiagramme erstellen, Sachen aus den Issues in der Wiki ablegen, evtl. Stories schreiben, Phasen(Zeit-)plan erstellen

**Reply by culmets** [reply]
Ja also wenn du jetzt auf die JPA Geschichte raus willst, das mach ich bzw. würde ich machen. Die SQL Abfragen hat man da ja in einer Java Klasse die irgendwas-Repo heißt. @MatejEvc
